spyder/config/manager.py

```diff
@@ -605,8 +605,9 @@ def config_shortcut(self, action, context, name, parent):
     def iter_shortcuts(self):
         """Iterate over keyboard shortcuts."""
         for context_name, keystr in self._user_config.items('shortcuts'):
-            context, name = context_name.split('/', 1)
-            yield context, name, keystr
+            if 'additional_configuration' not in context_name:

```

Comment:
Is this something that it's required now for all third-party plugins? I mean, the `additional_configuration` part in `context_name`?

Reply:
It is because the additional conf options add that string to the list, and I didn't found a way to avoid this so the safest is assume that is not a shortcut

Reply:
What are the additional conf options? Could you give an example?

Reply:
If a plugin were to include new configuration options to `shortcuts`, it will introduce the `additional_configuration` key on the configuration. This is added when a plugin is registered on the preferences one: 

See https://github.com/spyder-ide/spyder/blob/3f7aba19d90c6e873b639ff5420afd642961494c/spyder/api/plugins.py#L703 and https://github.com/spyder-ide/spyder/blob/3f7aba19d90c6e873b639ff5420afd642961494c/spyder/plugins/preferences/plugin.py#L134

Reply:
Ok, thanks for the explanation. It's clearer for me now.